### PR TITLE
Make tweaks to academia cta message

### PR DESCRIPTION
**Pull request by Shamarni96**
### Description
made design tweaks for the academiaCTA message.

- [x] Align the link text vertically in the box
- [x] Change font size of header to 24px
- [x] Change font size of the body copy to 20px
- [x] Make the image left-aligned, with 60px padding between text and image (desktop).
- [x] 24px top and right padding for the close icon (desktop and mobile).

### Screenshots

| Before | After |
| ------ | ----- |
| Screenshot 2022-06-23 at 15 51 33|  Screenshot 2022-06-23 at 15 51 46     |

### Ticket
https://financialtimes.atlassian.net/browse/UG-911

### CODE REVIEW

**Review by ivomurrell — APPROVED** [review]

File: client/components/bottom/academia-cta/main.scss [hunk 1]
```diff
@@ -16,7 +21,7 @@
 	}
 
 	&-heading h1 {
-		font-size: 20px;
```
Comment (jamesnicholls) [inline client/components/bottom/academia-cta/main.scss:19]: It's the desktop (line 30) that needs to change to 24px font size

File: client/components/bottom/academia-cta/main.scss [hunk 2]
```diff
@@ -27,7 +32,7 @@
 	}
 
 	&-body-text {
-		font-size: 18px;
```
Comment (jamesnicholls) [inline client/components/bottom/academia-cta/main.scss:30]: It's the desktop (line 44) that needs to change to 20px

**Review by jamesnicholls — CHANGES_REQUESTED** [review]
The changes to the font sizes need to be applied at desktop (large) screen sizes.

**Review by jamesnicholls — APPROVED** [review]
Looks great, thanks.

**Review by jenniferemshepherd — APPROVED** [review]
Top work 🥳

**Pull request merged**